### DISCUSSION TITLE
Scala Helpers

### DISCUSSION THREAD

**Discussion by hdost** [open-telemetry/opentelemetry-java#3420] [Q&A]
I'm not sure if there's any interest in expanding the utility of Java into scala. I know that there can be helpers written sometime to make things more scala friendly. I thought I might put it out there in case there are others interested in using scala with OTel.

**Comment by jkwatson** [comment]
I think this would be great. I'm not sure if would belong in this repository (we do have a little kotlin context sugar already here) or in the contrib repository. Either way, this would be a great addition to the opentelemetry ecosystem.
